house/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Country(models.Model):
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class City(models.Model):
+    country = models.ForeignKey(Country, on_delete=models.CASCADE)
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class Job(models.TextChoices):
+    TEACHER = 'Teacher'
+    STUDENT = 'Student'
+    PROGRAMMER = 'Programmer'
+    ARTIST = 'Artist'
+    MANAGER = 'Manager'
+    ARMY = 'Army'
+    POLICE = 'Police'
+    DOCTOR = 'Doctor'
+    VET = 'Vet'
+    NURSE = 'Nurse'
+    TECHNICHIAN = 'Technichian'
+    CLEANER = 'Cleaner'
+    OTHER = 'Other'
+    UNEMPLOYED = 'Unemployed'
+
+
+class House(models.Model):
```

Comment:
I think we should use Django default for PK in all the cases we are using something unique in any way
It will save us handling with creating a new unique id for each of our entities or giving them some UUID

Reply:
@LiorKGOW @shlior7 @ronyyosef was this resolved?